bench/src/main.rs

```diff
@@ -131,7 +129,7 @@ async fn main() {
 
     let avg_metric = Metric::from(avg_metric);
 
-    info!("Avg Metric {avg_metric:?}",);
+    info!("Avg Metric {:?}", avg_metric);
```

Comment:
wondering what's wrong with the old style 🤔

Reply:
did not know it is old ;) my IDE does not offer the same commands. I always used the latter style? why are you advocating for the former?

Reply:
reverted ..